src/OpenLoco/src/Entities/Entity.h

```diff
@@ -47,9 +59,9 @@ namespace OpenLoco
         EntityId llPreviousId;    // 0x06
         uint8_t linkedListOffset; // 0x8
         uint8_t var_09;
-        EntityId id; // 0xA
-        uint16_t var_0C;
-        Map::Pos3 position; // 0x0E
+        EntityId id;               // 0xA
+        VehicleFlags vehicleFlags; // Move these to VehicleBase after full reimplementation
```

Comment:
could you add 0xC just so we know what the offset was. It sometimes helps when RE'ing a function.